sources/packages/backend/apps/api/src/services/notification/gc-notify-actions.service.ts

```diff
@@ -60,9 +77,17 @@ export class GCNotifyActionsService {
         date: this.getDateTimeOnPSTTimeZone(),
       },
     };
-    return this.gcNotifyService.sendEmailNotification<MinistryStudentFileUploadPersonalisation>(
+
+    // Save notification into notification table.
+    const notificationSaved = await this.notificationService.saveNotification(
+      userId,
+      NotificationMessageType.MinistryFileUpload,
+      MINISTRY_FILE_UPLOAD_TEMPLATE_ID,
```

Comment:
Moving this to the "notification_messages" table will not be handled in this PR?

Reply:
I will update.

Reply:
Updated.